fetcher/account_test.go

```diff
@@ -63,6 +63,7 @@ func TestAccountBalanceRetry(t *testing.T) {
 		expectedBlock       *types.BlockIdentifier
 		expectedAmounts     []*types.Amount
 		expectedError       error
+		unretriableError    bool
```

Comment:
`nonretriable` vs. `unretriable`? not sure which is correct though

Reply:
honestly it should just be `retriable` but I was being lazy...one sec